src/components/Editor/editor.ts

```diff
@@ -0,0 +1,65 @@
+import { CodeJar } from 'codejar';
+import { withLineNumbers } from 'codejar/linenumbers';
+
+export default function editor(node: HTMLElement, { code, schema, autofocus = false, loc = false, ...options }) {
+    const highlight = (node: HTMLElement) => {
+        const content = node.textContent || code
+        const strings = content.split('\n')
+        const defaults = ['end']
+        const keys = [...getKeys(schema), ...defaults]
+
+        node.children[0].innerHTML = `${strings
+            .map((string: string) => string.split(' ').map((word: string) => mark(keys, word)).join(' '))
+            .join('\n')}`
```

Comment:
@Valexr can you see the code scanning warning by github on this line?

Reply:
This is temporally solution to use code source from `<pre><slot name="code" /></pre>`& now we dont need this. 
I'm remove it because escaping meta-har from any lang types very big task 😁